desktop/updater/auto-updater/index.js

```diff
@@ -5,58 +5,68 @@
  */
 const EventEmitter = require('events').EventEmitter;
 const util = require('util');
-const electron = require( 'electron' );
-const autoUpdater = electron.autoUpdater;
-const dialog = electron.dialog;
+const { dialog } = require('electron');
+const { autoUpdater } = require('electron-updater');
+const sanitizeHtml = require('sanitize-html');
+const TurndownService = require('turndown');
```

Comment:
I wondered if we could do the HTML to MD conversion with `showdown` (the MD converter we're already using), but it looks like the feature [hasn't shipped yet](https://github.com/showdownjs/showdown/issues/582) 🤷‍♀️ We'll replace it when it's released.